cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -412,8 +443,8 @@ subroutine evp (dt)
                          indxti      (:,iblk), indxtj      (:,iblk), & 
                          indxni      (:,iblk), indxnj      (:,iblk), & 
                          aiN       (:,:,iblk), nmass     (:,:,iblk), & 
-                         nmassdti  (:,:,iblk), fcorN_blk  (:,:,iblk), & 
-                         nmask     (:,:,iblk),                       & 
+                         nmassdti  (:,:,iblk), fcorN_blk  (:,:,iblk),& 
+                         nmask     (:,:,iblk),                       &                      
```

Comment:
can we avoid unnecessary whitespace changes ? this introduces false positive in 'git blame' output, if care is not taken..,.

Reply:
changed